spec/features/users/set_organisation_spec.rb

```diff
@@ -33,13 +18,6 @@
     then_the_users_organisation_name_is_updated
   end
 
-  scenario "Super admin can change their own organisation" do
-    given_i_am_a_super_admin_in_the_government_digital_service_organisation
-    when_i_change_my_organisation_to_test_org
-    then_i_cannot_see_the_government_digital_service_forms
-    but_i_can_see_the_test_org_forms
-  end
```

Comment:
We felt these tests were unnecessary?

Reply:
yeah sorry we should have commented on this. We decided this wasn't really a very useful test to have as there is nothing special about changing organisation and being super admin.

Within groups, we already test that super admin can see other organisation groups.